frontend/src/metabase/public/components/widgets/SharingPane/SharingPane.tsx

```diff
@@ -69,56 +70,59 @@ function SharingPane({
   );
 
   return (
-    <Group p="lg" data-testid="sharing-pane-container">
-      <SharingPaneButton
-        header={t`Static embed`}
-        description={t`Securely embed this dashboard in your own application’s server code.`}
-        illustration={<StaticEmbedIcon />}
-        onClick={() => onChangeEmbedType("application")}
-      >
-        <SharingPaneActionButton
-          data-testid="sharing-pane-static-embed-button"
-          fullWidth
+    <Stack p="lg">
```

Comment:
The diff looks weird but I just nested everything into the `Stack` here and added the `InteractiveEmbeddingCTA` at the bottom